test/unittests/evm_other_test.cpp

```diff
@@ -56,3 +56,22 @@ TEST_F(evm_other, loop_full_of_jumpdests)
5620076fa5760006000fd5b80830192505050806000018035600160200383360303811215156200771f5760006000fd5b8083019250508135905060208201915067ffffffffffffffff811115620077465760006000fd5b6020810236038213156200775a5760006000fd5b600081811015156200776857fe5b90506020028101803560016080038336030381121515620077895760006000fd5b8083019250505080606001803560016020038336030381121515620077ae5760006000fd5b8083019250508135905060208201915067ffffffffffffffff811115620077d55760006000fd5b602081023603821315620077e95760006000fd5b60018181101515620077f757fe5b90506020028101803560016020038336030381121515620078185760006000fd5b8083019250508135905060208201915067ffffffffffffffff8111156200783f5760006000fd5b602081023603821315620078535760006000fd5b600381811015156200786157fe5b90506020028101803560016020038336030381121515620078825760006000fd5b8083019250508135905060208201915067ffffffffffffffff811115620078a95760006000fd5b602081023603821315620078bd5760006000fd5b9050141515620078d1576026905062020e6a565b7f6e73460df9fa5214e3ef23dd4259ca09493a35940000000000000000000000008280600001803560016020038336030381121515620079115760006000fd5b8083019250508135905060208201915067ffffffffffffffff811115620079385760006000fd5b6020810236038213156200794c5760006000fd5b600081811015156200795a57fe5b905060200281018035600160600383360303811215156200797b5760006000fd5b8083019250505080600001803560016020038336030381121515620079a05760006000fd5b8083019250508135905060208201915067ffffffffffffffff811115620079c75760006000fd5b602081023603821315620079db5760006000fd5b60008181101515620079e957fe5b9050602002810180356001602003833603038112151562007a0a5760006000fd5b8083019250508135905060208201915067ffffffffffffffff81111562007a315760006000fd5b60208102360382131562007a455760006000fd5b6000818110151562007a5357fe5b9050602002810180356001602003833603038112151562007a745760006000fd5b808301925050508060000180356001602003833603038112151562007a995760006000fd5b8083019250508135905060208201915067ffffffffffffffff81111562007ac05760006000fd5b60208102360382131562007ad45760006000fd5b6000818110151562007ae257fe5b9050602002810180356001608003833603038112151562007b035760006000fd5b808301925050508060600180356001602003833603038112151562007b285760006000fd5b8083019250508135905060208201915067ffffffffffffffff81111562007b4f5760006000fd5b60208102360382131562007b635760006000fd5b6001818110151562007b7157fe5b9050602002810180356001602003833603038112151562007b925760006000fd5b8083019250508135905060208201915067ffffffffffffffff81111562007bb95760006000fd5b60208102360382131562007bcd5760006000fd5b6003818110151562007bdb57fe5b9050602002810180356001602003833603038112151562007bfc5760006000fd5b8083019250508135905060208201915067ffffffffffffffff81111562007c235760006000fd5b60208102360382131562007c375760006000fd5b6000818110151562007c4557fe5b905060200201602062007c5c919081019062031fce565b6bffffffffffffffffffffffff191614151562007c7d576027905062020e6a565b7fbcba2c46958e61fa2fb4042cd7bc5816a710195b000000000000000000000000828060000180356001602003833603038112151562007cbd5760006000fd5b808301925050813590506020" +
```

Comment:
The reason the large string was split is because the clang compiler errors on string literals larger than 65535 bytes. However, what I'm doing here is also wrong. I need to figure out a way to populate code with smaller string literal fragments